initrd/bin/oem-factory-reset

```diff
@@ -425,356 +833,464 @@ $TPM_STR
           * Sign all of the files in /boot with the new GPG key\n\n
         It requires that you already have an OS installed on a\n
         dedicated /boot partition. Do you wish to continue?" \
-        $HEIGHT $WIDTH $CONTINUE $CANCEL $CLEAR $bg_color --title "$title_text" ; then
+    $HEIGHT $WIDTH $CONTINUE $CANCEL $CLEAR $bg_color --title "$title_text"; then
     exit 1
 fi
 
+#Make sure /boot is mounted if board config defines default
+mount_boot
 # We show current integrity measurements status and time
 report_integrity_measurements
 
 # Determine gpg algorithm to be used, based on available usb-token
 usb_security_token_capabilities_check
 
-use_defaults=n
-if [ "$CONFIG_OEMRESET_OFFER_DEFAULTS" = y ]; then
-	echo -e -n "Would you like to use default configuration options?\nIf N, you will be prompted for each option [Y/n]: "
-	read -n 1 use_defaults
+# Clear the screen
+clear
+
+use_defaults=y
+if [ "$CONFIG_OEMRESET_OFFER_DEFAULTS" = y -o "$use_defaults" = y ]; then
```

Comment:
This `if` is always taken now, the condition is always true.  Please delete it and delete CONFIG_OEMRESET_OFFER_DEFAULTS from board configs

Reply:
@JonathonHall-Purism done under [637ebcf](https://github.com/linuxboot/heads/pull/1515/commits/637ebcfc1c6578c4c22b7f12cfc8495a58cd53e3)